routes/web.php

```diff
@@ -218,6 +219,13 @@
         Route::post('ticket/blacklist/delete/{id}', [ModTicketsController::class, 'blacklistDelete'])->name('ticket.blacklist.delete');
         Route::post('ticket/blacklist/change/{id}', [ModTicketsController::class, 'blacklistChange'])->name('ticket.blacklist.change');
         Route::get('ticket/blacklist/datatable', [ModTicketsController::class, 'dataTableBlacklist'])->name('ticket.blacklist.datatable');
+
+
+        Route::get('ticket/category', [TicketCategoryController::class, 'index'])->name('ticket.category.index');
+        Route::get('ticket/category/datatable', [TicketCategoryController::class, 'datatable'])->name('ticket.category.datatable');
+        Route::post('ticket/category', [TicketCategoryController::class, 'store'])->name('ticket.category.store');
+        Route::delete('ticket/category/destroy/{id}', [TicketCategoryController::class, 'destroy'])->name('ticket.category.destroy');
```

Comment:
Isnt destroy a base method that will laravel expose as a http DELETE route anyways?

Reply:
Idk. He didnt like a Post request. Even if So, dosent harm having it as delete Route does it?